src/lib/roles.ts

```diff
@@ -0,0 +1,176 @@
+import { get } from 'svelte/store';
+import {
+	gameStore,
+	type GameRole,
+	type GameRoleName,
+	type Role,
+	type RoleName,
+	type UtilityRole,
+	type UtilityRoleName
+} from './stores/game';
+
+const werewolfsCount = () => getGameRole('werewolf')?.amount ?? 0;
+
+const normalCount = (roles: Role[]) =>
+	roles
```

Comment:
Indentation is a bit off

Reply:
seems like this is the way prettier wants it